akka-docs-new/src/main/paradox/guide/quickstart.md

```diff
@@ -0,0 +1,113 @@
+For our sample project, this file should contain the following:
+
+```scala
+// build.sbt
+
+name := "intro-akka"
+organization := "intro-akka.organization"
+version := "0.1-SNAPSHOT"
+
+scalaVersion := "2.11.8"
+val AkkaVersion = "2.4.12"
+
+libraryDependencies += "com.typesafe.akka" %% "akka-actor" % AkkaVersion
+libraryDependencies += "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test"
+```
+
+This simple file sets up first the project metadata (_name_ and _organization_; we just picked a sample one here). 
+Then we set up the version of the scala compiler we use, then set a variable with the Akka version we intend to
+use (always try to use the latest). 
+
+As the last step, we add two dependencies, `akka-actor` and `akka-testkit`. Note that we used the `AkkaVersion` 
+variable for both dependencies, ensuring that versions are not accidentally mixed and kept in sync when upgrading.
+
+Finally, check that everything works by running `sbt update` from the base directory of your project 
+(where the `build.sbt` file is).
+
+## Setting up your IDE
+
+If you go with IDEA, you usually have flexible means to import project either manually created by one of the 
+previous steps from Setting up Your Build, or to let it create it for you. Depending on your build tool, 
+there are detailed steps in the IDEA documentation
+
+sbt
+:  [https://www.jetbrains.com/help/idea/2016.2/getting-started-with-sbt.html](https://www.jetbrains.com/help/idea/2016.2/getting-started-with-sbt.html)
+
+## Building the first application
+
+Akka applications are simply Scala/Java applications, you don’t need to set up any container (application server, etc.) 
+to have an actor system running. All we need is a class with a proper `main` method that stops and starts an actor 
+system. The pieces of the puzzle that we need to put together are: What is an actor system and why do I need one? 
+How can I start and stop it? Why do I need to stop it?
+
+In Akka, actors belong to actor systems, which are instances of the type `ActorSystem`. This class acts as a 
+resource container which holds among others
+
+* configuration shared by all actors in that system
+* a pool of threads that will execute actors that are ready to process messages
+* a dispatcher mechanism that dynamically assigns actors to threads of the pool
+* a scheduler used for timer related tasks
+
+The `ActorSystem` manages its actors and runs them in the background on its encapsulated thread pool. 
+This means that is must be explicitly shut down, otherwise these threads keep running and the JVM does 
+not exit (by default threads created by ActorSystem are not daemon threads; see the JDK documentation on 
+more details on [daemon or non-daemon threads](https://docs.oracle.com/javase/8/docs/api/java/lang/Thread.html)). 
+
+This is the usual pattern to have your system set up and stopped on external signal 
+(user pressing ENTER in the console):
+
+
```

Comment:
It is in the original google doc. I didn't continue porting it because it is likely superseded by g8 now.